### PR TITLE
Bunch of grammar-related changes to xCal/xCard post

### PR DESCRIPTION
**Pull request by evert**
I tried to stay true to what you wanted to say, mostly keep your writing style in-tact, but improve a few sentence structures :) Hope it helps, nice post!

### CODE REVIEW

**Comment by Hywan** [comment]
Thank you very much! Help me to improve my English!

File: Application/In/Posts/2015-02-27_xCal_xCard_and_RFC_errata.xyl [hunk 1]
```diff
@@ -53,7 +52,7 @@ UID:4088E990AD89CB3DBB484909
 END:VEVENT
 END:VCALENDAR</code></pre>
     <p>Horrible, yes. You were warned. These formats are standardized in several
-    RFC, to list some of them:
+    RFCs, to list some of them:
```
Comment (Hywan) [inline Application/In/Posts/2015-02-27_xCal_xCard_and_RFC_errata.xyl]: Should we pluralize abbreviations in English?

**Comment by Hywan** [comment]
Thanks!
I changed the title too: http://mnt.io/P/2015-02-27_RFCs_should_provide_executable_test_suites.html.

File: Application/In/Posts/2015-02-27_xCal_xCard_and_RFC_errata.xyl (hunk 1 above)
Comment (evert) [inline Application/In/Posts/2015-02-27_xCal_xCard_and_RFC_errata.xyl]: Yes, definitely. Some people use an apostrophe (RFC's) but I think RFCs is more correct.

**Pull request merged**